framework/components/ecl-dropdowns/_dropdowns.scss

```diff
@@ -0,0 +1,22 @@
+/**
+ * ECL Dropdowns
+ * @define dropdowns
+ */
+
+.ecl-dropdowns {
+  position: relative;
+
+  // collapsed/open part of the dropdown
+  &__body {
```

Comment:
Personally, I don't like writing thinks like `&__something` because it makes the code hard to search through. I invite you to read https://csswizardry.com/2017/02/code-smells-in-css-revisited/#string-concatenation-for-classes and to watch the screencast :)